### PR TITLE
Update README and license

### PR DESCRIPTION
**Pull request by mrodem**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,72 @@
...
-* [pc-ble-driver  library](https://github.com/NordicSemiconductor/pc-ble-driver)
- 
+pc-ble-driver-js is a Node.js AddOn for the [pc-ble-driver  library](https://github.com/NordicSemiconductor/pc-ble-driver). It depends on the pc-ble-driver repository referenced as a submodule.
 
-The AddOn does not support all of the functions in the [pc-ble-driver  library](https://github.com/NordicSemiconductor/pc-ble-driver).
-
-This is a very early implementation, we are working on making the AddOn production quality.
-
-When production quality is reached we will probably publish it on npmjs. Stay with us!
+The AddOn does not support all of the functions in the pc-ble-driver library. This is a very early implementation, we are working on making the AddOn production quality. When production quality is reached we will probably publish it on npmjs. Stay with us!
```
Comment (bihanssen) [inline README.md]: This paragraph can be removed

File: README.md [hunk 2]
```diff
@@ -1,53 +1,72 @@
...
-## Ubuntu Linux
+# Examples/system tests
 
-Setup npm:
+Examples/system tests require one or more development kits to be connected to your computer. These tests can be found in the `test` directory, and have to be run manually. F.ex. to test scanning, run:
 
-```
-npm config set cmake_BOOST_ROOT=/path/to/boost_x_xx_x
-```
+    node test/simpleScanTest.js
```
Comment (bihanssen) [inline README.md]: Add section on contributions/CLA

File: README.md (hunk 1 above)
Comment (mrodem) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (mrodem) [inline README.md]: Done

**Pull request merged**